loctool.js

```diff
@@ -83,6 +83,10 @@ function usage() {
         "  Specify the dir where the xliffs files live. Default: \".\"\n" +
         "--localizeOnly\n" +
         "  Generate a localization resource only. Do not create any other files at all after running loctool. \n" +
+        "--xliffResRoot\n" +
+        "  Specify the dir where the xliffs resource generation output (Default is resources/) live.. \n" +
+        "--xliffResName\n" +
+        "  Specify the resource filename from xliff resource generation (Default is strings.json/) live.. \n" +
```

Comment:
Make this `Specify the resource filename used during resource file generation. (Default is strings.json/)`

Reply:
Update it. Thanks!